can_tools/scrapers/official/CT/ct_state.py

```diff
@@ -0,0 +1,198 @@
+from typing import Any
+
+import pandas as pd
+import us
+import requests
+
+from can_tools.scrapers.base import CMU, DatasetBase
+from can_tools.scrapers.official.base import SODA
+
+
+class CTCountyDeathHospitalizations(SODA, DatasetBase):
+    source = "https://data.ct.gov/bfnu-rgqt.json"
+    state_fips = int(us.states.lookup("Connecticut").fips)
+    baseurl = "https://data.ct.gov/"
+    has_location = False
+    location_type = "county"
+    cdh_id = "bfnu-rgqt"
+    state_fips = int(us.states.lookup("Connecticut").fips)
+    source = "https://data.ct.gov/resource/bfnu-rgqt.json"
+
+    def fetch(self):
+        cdh = self.get_dataset(self.cdh_id)
+
+        return cdh
+
+    def normalize(self, data) -> pd.DataFrame:
+        # fips_df = pd.DataFrame.from_dict(self.state_fips)
+        cdh_rename = {
```

Comment:
See comment below about renaming columns AND using custom names as keys of `crename` later